allennlp_models/tagging/models/crf_tagger.py

```diff
@@ -296,6 +296,12 @@ def decode_top_k_tags(top_k_tags):
 
         return output_dict
 
+    @overrides
+    def cuda(self, device=None):
+        super().cuda(device)
+        self.crf.cuda(device)
+        return self
```

Comment:
Is this really necessary? `self.crf` is a `nn.Module`, so I would think this already works fine without overriding this method.

Reply:
@epwalsh you were true, just checked it. We can safely remove this change. The bug is only in the `conditional_random_field.py`

Reply:
Fixed: https://github.com/allenai/allennlp/pull/5335/commits/84d4e20282ef05a16762cf322683e5b3512db8ab